db/sql/3_msar_custom_aggregates.sql

```diff
@@ -37,3 +37,61 @@ CREATE OR REPLACE AGGREGATE peak_time (TIME)
     finalfunc = final_func_peak_time,
     initcond = '{0,0}'
 );
+
+
+CREATE OR REPLACE FUNCTION dow_to_degrees(_date DATE)
+	returns DOUBLE PRECISION AS $$
+    SELECT (EXTRACT(DOW FROM _date)::double precision) * 360 / 7;    
+$$ LANGUAGE SQL;
+
+
+CREATE OR REPLACE FUNCTION degrees_to_dow(degrees DOUBLE PRECISION)
+	returns INT AS $$
+    SELECT (ROUND(degrees * 7 / 360)::int) % 7;
+$$ LANGUAGE SQL;
+
+
+
+CREATE OR REPLACE FUNCTION dow_to_string(_dow INT)
```

Comment:
Is `dow` a keyword? If not, use that for the variable name instead. If so, use `dow_`.